input/fsh/models.fsh

```diff
@@ -79,6 +79,21 @@ criteria are defined by FHIRPath expressions.
   * description 0..1 markdown "Description of the column" """
     A human-readable description of the column.
   """
+  * collection 0..1 boolean "Indicates whether the column may have multiple values." """
+  Indicates whether the column may have multiple values. Defaults to `false` if unset.
+  
+  ViewDefinitions must have this set to `true` if multiple values may be returned. Implementations should
+  report an error if multiple values are produced when that is not the case.
+  """
+  * type 0..1 uri "A FHIR StructureDefinition URI for the column's type." """
+  A FHIR StructureDefinition URI for the column's type. Relative URIs are implicitly given
+  the 'http://hl7.org/fhir/StructureDefinition/' prefix. The URI may also be a content reference to a
+  backbone element of a containing FHIR type. For instance, `Observation#referenceRange` may be specified to indicate
```

Comment:
It may be more common/understandable to use element ID notation like `Observation.referenceRange` for core resources.

In cases where there a structured definition has multiple slices for the same path, again the IDs diaambiguate, like `Observation.component:SystolicBP` from http://hl7.org/fhir/StructureDefinition/bp

Or maybe I'm just not understanding what logic to use to determine what goes after the `#` in the proposed syntax.